universe-ai/universe
# Refactor online nodes certs

+ Add onlineValidation to node

+ Add online revoke status to nodes and certs

* Refactor network to onlineIdNetwork and remove config flag

* Refactor BaseCert and DataModelInterface from dynamic to online

* Adjust all cert expected max sizes

- Remove dynamicSelfSpec property from BaseCert (dynamic/online nodes)
